doc/installing-seedtool.md

```diff
@@ -0,0 +1,145 @@
+  - In macOS this will be in `~/Documents/Arduino`
+
+### Preparing the *Arduino IDE* for building *LetheKit*
+
+You'll need to add files to the *Arduino IDE* to support the [SAMD51 Thing Plus](https://www.sparkfun.com/products/14713) board used by *LetheKit*.
+
+- In the Arduino app, open **Preferences** (**Arduino** > **Preferences**), find the  **Additional Board Manager URLs** text box, and copy this  URL to it and click OK.
+  -  `https://raw.githubusercontent.com/sparkfun/Arduino_Boards/master/IDE_Board_Manager/package_sparkfun_index.json1`
+- Now we need to install the correct Arduino SAMD Core. Navigate to your **Board Manager** (**Tools** > **Board**: > **Boards Manager...**), then find an entry for **Arduino SAMD Boards (32-bits ARM Cortex-M0+)**. It may help to enter `SAMD` into the search bar. Select that Arduino core and install the latest version (as of the creation of this document _v1.8.6_).
+- Now we need to install the **SparkFun SAMD** Add-on. Enter `sparkfun samd` in the **Board Manager**'s search bar, which should show **SparkFun SAMD Boards** with a dependency to the SAMD Core you just installed. Install this (as of the creation of this document *v1.7.4*).
+- Now select from the **Tools** > **Board**: > list, choose **SparkFun Thing Plus**. Now when you look at the **Tools** menu you should see this board selected as default.
+- Now you need to select the serial port. Connect the LetheKit to your computer and turn it on, and then navigate back up to the **Tool** > **Port** menu. The port menu should magically know which of your ports (if you have more than one) is the SAMD51 Thing Plus board.
+  - On Linux, the port should look something like ??
+  - In macOS the port should look something like "**/dev/cu.usbmodem####**", not a bluetooth port.
+- Next we need to install the various hardware libraries for *LetheKit*. Navigate to your Library Manager (**Tools** > **Manage Libraries…**).
+
+  - Display Library: Enter `GxEPD2` in the Library Manager's search bar, which should show **GxEPD2** library. Install this (as of the creation of this document *v1.2.6*), and all dependencies.
+  - Keypad: Enter `Keypad` in the Library Manager's search bar, which should show a number of libraries, but only one is called just **Keypad** library. Install this (as of the creation of this document *v3.1.1*).
+
+### Preparing LetheKit Libraries for installation
+
+*LetheKit* provides a collection of security submodules (libraries) that work well together. You install them into your local *Arduino* development environment and use them to build the provided sample *Seedtool* application or develop your own applications before uploading them to your SAMD51 board used by *LetheKit*
+
+The LetheKit software can be placed anywhere you like. It does not need to be inside your Arduino sketchbook. These instructions we'll presume you want to install it in `src/bc-lethekit` inside your default Arduino directory.
```

Comment:
It does **not** need to be in the default Arduino directory.  I suppose it might work there, but it might cause other issues since the Arduino IDE is nosy and might feel it should do something to/about the tree.  I guess if it's working for you it's ok ...

Reply:
It does work fine for me there at least on Mac.